### PR TITLE
Add 32px mod icons to the images dir.

### PR DESCRIPTION
**Pull request by pchote**
This PR introduces an images/icons/ dir with 32 and 64 px icons for the default mods.  The 64 px versions are used by the about page (and the remaining images moved up a level), and the 32 px versions will be used as part of the extended mod metadata for the master server advertisements.

### CODE REVIEW

**Comment by pchote** [comment]
False positive, it will be fine on merge.

File: content/about.html [hunk 1]
```diff
@@ -33,10 +33,10 @@ <h3>What is OpenRA?</h3>
 
 <div class="about-modblock">
   <div class="about-modimage about-right">
-    <img width="312" height="175" src="/images/about/ingame-cnc.png" />
+    <img width="312" height="175" src="/images/ingame-cnc.png" />
   </div>
   <div class="about-modtext about-left" style="margin-top:10px">
-    <h3><img src="/images/about/mod-cnc-logo.png" width="32" height="32" />Tiberian Dawn</h3>
+    <h3><img src="/images/icons/cnc_64x64.png" width="32" height="32" />Tiberian Dawn</h3>
```
Comment (reaperrr) [inline content/about.html]: Why are these called _64x64?  #15185 references filenames with _32x32.

Comment (pchote) [reply content/about.html]: The about page uses the 64px versions to improve visuals on 2x HiDPI displays.  These are separate assets to the new 32px versions which are used by #15185 but not the website.

**Comment by pchote** [comment]
I'll merge this tonight if there aren't any objections.

**Pull request merged**